src/comestible_inv_listitem.cpp

```diff
@@ -0,0 +1,771 @@
+        nc_color &print_col )
+{
+    //Function is overriden in children. Code is to supress warnings/errors.
+    if( col == 0 ) {
+        print_str = "";
+        print_col = c_white;
+    }
+    return false;
+}
+
+bool comestible_inv_listitem::print_default_columns( comestible_inv_columns col,
+        std::string &print_str, nc_color &print_col )
+{
+    switch( col ) {
+        case COLUMN_VOLUME: {
+            bool it_vol_truncated = false;
+            double it_vol_value = 0.0;
+            print_str = format_volume( volume, 5, &it_vol_truncated, &it_vol_value );
+            if( it_vol_truncated && it_vol_value > 0.0 ) {
+                print_col = c_red;
+            } else {
+                print_col = volume.value() > 0 ? menu_color : menu_color_dark;
+            }
+            break;
+        }
+
+        case COLUMN_WEIGHT: {
+            double it_weight = convert_weight( weight );
+            size_t w_precision;
+            print_col = it_weight > 0 ? menu_color : menu_color_dark;
+
+            if( it_weight >= 1000.0 ) {
+                if( it_weight >= 10000.0 ) {
+                    print_col = c_red;
+                    it_weight = 9999.0;
+                }
+                w_precision = 0;
+            } else if( it_weight >= 100.0 ) {
+                w_precision = 1;
+            } else {
+                w_precision = 2;
+            }
+            print_str = string_format( "%5.*f", w_precision, it_weight );
+            break;
+        }
+        case COLUMN_AMOUNT: {
+            if( stacks > 1 ) {
+                print_col = menu_color;
+                if( stacks > 9999 ) {
+                    stacks = 9999;
+                    print_col = c_red;
+                }
+                print_str = string_format( "%4d", stacks );
+            } else {
+                print_str = "";
+            }
+            break;
+        }
+        case COLUMN_SRC:
+            print_col = menu_color;
+            print_str = area->info.get_shortname();
+            break;
+
+        default:
+            return false;
+            break;
+    }
+    set_print_color( print_col, print_col );
+    return true;
+}
+
+void comestible_inv_listitem::print_name( catacurses::window window,
+        int right_bound, int cur_print_y )
+{
+    std::string item_name;
+
+    item *it = items.front();
+    if( it->is_money() ) {
+        //Count charges
+        unsigned int charges_total = 0;
+        for( const auto item : items ) {
+            charges_total += item->charges;
+        }
+        item_name = it->display_money( items.size(), charges_total );
+    } else {
+        item_name = it->display_name();
+    }
+    //check stolen
+    if( it->has_owner() ) {
+        const faction *item_fac = it->get_owner();
+        if( item_fac != g->faction_manager_ptr->get( faction_id( "your_followers" ) ) ) {
+            item_name = string_format( "%s%s", colorize( "!", c_light_red ), item_name );
```

Comment:
```suggestion
            item_name = colorize( "!", c_light_red ) + item_name;
```
"%s%s" is simple string concatenation. It doesn't need string formatting.